### PR TITLE
Tta 4 fungible tokens receive token not shown

### PR DESCRIPTION
**Pull request by ryancwalsh**
https://near-foundation.atlassian.net/jira/software/c/projects/TTA/boards/5?modal=detail&selectedIssue=TTA-4

image

The main part of this PR is the addition of this SQL file, but I made a lot of other changes along the way while learning how it all works:

`server/src/helpers/TxTypes/Fungible tokens - Receive token.sql`
https://github.com/NEARFoundation/tx-tracking-app/compare/TTA-4-fungible-tokens-receive-token-not-shown?expand=1#diff-f1413a6e34e45fa605ca5106c5c8abb1719bb2a45ba491d0d87ca48038eccdcdR1

### CODE REVIEW

File: server/src/routes/collector.routes.js [hunk 1]
```diff
@@ -10,4 +11,5 @@ export const routes = (app) => {
   app.get('/types', getTypes);
   app.post('/accounts', getAccounts);
   app.post('/add-tasks', addTasks);
+  app.post('/deleteAccountData', deleteAccountData);
```
Comment (sandoche) [inline server/src/routes/collector.routes.js:14]: I would have put `'/delete-account-data'` to be consistent with the rest, what do you think?

Comment (sandoche) [reply server/src/routes/collector.routes.js:14]: I don't have the full picture of this issue, could you explain me the need of deleting account data?

**Review by sandoche — APPROVED** [review]

File: server/src/routes/collector.routes.js (hunk 1 above)
Comment (ryancwalsh) [reply server/src/routes/collector.routes.js:14]: @sandoche Cool, thanks for reviewing. I pushed a couple of commits based on your feedback. See below.

Comment (sandoche) [reply server/src/routes/collector.routes.js:14]: Awesome thanks!

**Pull request merged**